blocks/v2-testimonial/v2-testimonial.css

```diff
@@ -0,0 +1,174 @@
+/* stylelint-disable selector-class-pattern */
+
+.v2-testimonial {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
+
+.v2-testimonial__blockquote-column {
+  background-color: #2b2b2b;
+  padding: 48px 16px;
+  color: var(--c-primary-white);
+  font-family: var(--ff-body);
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 150%;
+  letter-spacing: 0.3px;
+  padding-right: calc(100vw - 510px);
+}
+
+.v2-testimonial__blockquote-column strong {
+  font-family: var(--ff-body-bold);
+}
+
+.v2-testimonial__heading {
+  color: var(--c-primary-white);
+  font-family: var(--ff-headline-medium);
```

Comment:
Let's use font variables defined here with font shorthand. Can remove font-style as it has no effect
` font: var(--headline-1-font-size) var(--ff-headline-medium);
   line-height: var(--headline-1-line-height);`
   
   Using short hand is optional

Reply:
Updated